src/libraries/System.Runtime/ref/System.Runtime.cs

```diff
@@ -11060,13 +11060,13 @@ public abstract partial class Assembly : System.Reflection.ICustomAttributeProvi
     {
         protected Assembly() { }
         [System.ObsoleteAttribute("Assembly.CodeBase and Assembly.EscapedCodeBase are only included for .NET Framework compatibility. Use Assembly.Location instead.", DiagnosticId = "SYSLIB0012", UrlFormat = "https://aka.ms/dotnet-warnings/{0}")]
-        [System.Diagnostics.CodeAnalysis.RequiresAssemblyFilesAttribute("The code will throw for assemblies embedded in a single-file app")]
+        [System.Diagnostics.CodeAnalysis.RequiresAssemblyFiles("This member throws an exception for assemblies embedded in a single-file app")]
```

Comment:
Here and in all ref updates shouldn't we put the complete name for the attribute? At least all other attributes are written in full